app/components/WorkExperience/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const SectionWrapper = styled.section`
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const SectionName = styled.h2`
+  width: 100%;
+  text-align: left;
+  font-size: 22px;
+`;
+
+const CompanyName = styled.h3`
+  font-size: 18px;
+`;
+
+const LeftColumn = styled.div`
+  width: 70%;
+  float: left;
+`;
+
+const RightColumn = styled.div`
+  width: 30%;
+  text-align: right;
+  float: right;
+`;
+
+function WorkingExperience(props) {
+  const { text, data } = props.dataSource;
```

Comment:
it is good to see you to use `spread` feature of ES6. The `dataSource` means a list of data. Let's look at the solution I provided.
The `WorkingExperience` component will have 2 props.
The first one is `title` and `dataSource`.

- title: Section name.
- dataSource: the list of items.

Here is how I use it
```js
<WorkingExperience title={title} dataSource={data} />
```
What do you think?